components/Modal/Modal.styled.ts

```diff
@@ -17,12 +17,15 @@ export const Background = styled.div`
     rgba(14, 16, 60, 0.6),
     rgba(14, 16, 60, 0.67)
   );
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
 `;
 
 export const ModalBox = styled(MaxWidth)`
   display: flex;
   flex-direction: column;
-  margin-top: 15vh;
+  margin-top: 24.5vh;
```

Comment:
This still has the same issue, you can keep it a static number for margin-top, otherwise it will always shift based on screen size.